template/.drone.star

```diff
@@ -35,7 +35,7 @@ def testing(ctx):
     'steps': [
       {
         'name': 'staticcheck',
-        'image': 'golang:1.13',
+        'image': 'golang:1.14',
```

Comment:
One maybe issue all the Windows stuff is stuck at 1.12

Reply:
Than we should really upgrade Go on Windows.

Reply:
Do you have access so we can?